tests/xharness/Harness.cs

```diff
@@ -284,6 +285,7 @@ public Harness (IResultParser resultParser, HarnessAction action, HarnessConfigu
 			SYSTEM_MONO = config ["SYSTEM_MONO"];
 			DOTNET_DIR = config ["DOTNET_DIR"];
 			INCLUDE_XAMARIN_LEGACY = config.ContainsKey ("INCLUDE_XAMARIN_LEGACY") && !string.IsNullOrEmpty (config ["INCLUDE_XAMARIN_LEGACY"]);
+			DotNetTfm = config ["DOTNET_TFM"];
```

Comment:
minor: I get itchy when I see raw strings like that. Would prefer to see consts in one place that have that or if you renamed the property to `DOTNET_TFM` you can use one of my favorite hacks, `nameof`:
```
DOTNET_TFM = config [nameof (DOTNET_TFM)];
```

Reply:
Agreed, but that's a bigger refactoring (the same should be done in the surrounding code as well).